src/main/java/controller/InputView.java

```diff
@@ -0,0 +1,54 @@
+package controller;
+
+import parser.InputParser;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    private final Scanner scanner = new Scanner(System.in);
+    private static final InputParser inputParser = new InputParser();
+
+    public String inputCalculationString() {
+        System.out.println("계산할 문자열을 입력하세요.(단, 숫자와 사칙 연산 사이에는 반드시 빈 공백 문자열이 있어야 합니다.)");
+
+        String calculateString = scanner.nextLine();
+
+        if (isValidInput(calculateString)) {
+            return calculateString;
+        }
+
+        return null;
```

Comment:
`null`을 반환하는 건 이후에 호출되는 로직에서 `NullPointerException`이 발생할 여지가 있기 때문에 굉장히 위험합니다.
`isValidInput()`을 거쳐서 예외가 발생하지 않는다면 그대로 `calculateString`을 반환하도록 해보면 좋을 것 같아요.

```suggestion
        isValidInput(calculateString);
        return calculateString;
```